lib/compiler/src/engine/trap/frame_info.rs

```diff
@@ -207,7 +203,7 @@ pub fn register(
     ) in finished_functions.iter()
     {
         let start = **start as usize;
-        let end = start + len;
+        let end = start + len - 1;
```

Comment:
Can we add a code comment here making explicit why is `-1` so we remember it easily for the future?